profiles/base.yaml

```diff
@@ -1144,6 +1144,12 @@ grave_junk:
 - millipede
 - cockroach
 
+#HE tarantula gift, list the skills you want the tarantula to use
+spider_noun: bio tarantula
+crafting_skills:
```

Comment:
❓ `*_skills` is really generic. What do you think about `spider_*_skills`?

Reply:
Sure can. I am testing some other pieces and why I have lag.